clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -486,8 +486,8 @@ public boolean matches(AbstractRequest body) {
         assertFalse(coordinator.rejoinNeededOrPending());
         assertEquals(toSet(assigned), subscriptions.assignedPartitions());
         assertEquals(subscription, subscriptions.groupSubscription());
-        assertEquals(1, rebalanceListener.revokedCount);
-        assertEquals(getRevoked(owned, assigned), rebalanceListener.revoked);
+        assertEquals(0, rebalanceListener.revokedCount);
+        assertNull(rebalanceListener.revoked);
```

Comment:
No that's because we check if the partition list if empty or not now: if it is empty, we'd skip it.

But after discussing @ableegoldman and @hachikuji on the other pt.2 PR I've decided to revert this and maintain the old behavior.